angr/exploration_techniques/loop_seer.py

```diff
@@ -46,7 +48,7 @@ def __init__(self, cfg=None, functions=None, loops=None, bound=None, bound_reach
                     self.loops[loop.entry_edges[0][0].addr] = loop
 
         elif loops is not None:
-            raise TypeError('What type of loop is it?')
+            raise AngrExplorationTechniqueError("Invalid type for 'loops' parameter!")
```

Comment:
Nitpick: I prefer raising a `TypeError` here, because it's an exception that is caused by a programming mistake, and it should catch developers' attention during development as early as possible. It _should not_ be caught-and-ignored by our resiliency support.